clients/ui/frontend/src/app/pages/modelCatalog/utils/validatedModelUtils.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  CatalogPerformanceMetricsArtifact,
+  CatalogAccuracyMetricsArtifact,
+} from '~/app/modelCatalogTypes';
+
+export type ValidatedModelMetrics = {
+  accuracy: number;
+  hardware: string;
+  hardwareCount: string;
+  rpsPerReplica: number;
+  ttftMean: number;
+};
+
+export const extractValidatedModelMetrics = (
+  performanceMetrics?: CatalogPerformanceMetricsArtifact,
+  accuracyMetrics?: CatalogAccuracyMetricsArtifact,
+): ValidatedModelMetrics => ({
+  accuracy: accuracyMetrics?.customProperties.overall_average?.double_value || 53.9,
+  hardware: performanceMetrics?.customProperties.hardware?.string_value || '8xH100-80',
+  hardwareCount: performanceMetrics?.customProperties.hardware_count?.int_value || '8',
+  rpsPerReplica: performanceMetrics?.customProperties.requests_per_second?.double_value || 1,
+  ttftMean: performanceMetrics?.customProperties.ttft_mean?.double_value || 1428,
+});
```

Comment:
This will need to be refactored a bit since you need to use multiple accuracy metrics artifacts (to average their accuracy) and multiple performance metrics artifacts (to show details from one at a time). I'd maybe split this utility in two (one for each artifact type) so you can use them independently. We should also add some basic unit tests for them.